commercetools-sdk-base/src/main/java/io/sphere/sdk/client/SphereClientConfig.java

```diff
@@ -78,6 +80,11 @@ public String getProjectKey() {
         return projectKey;
     }
 
+
+    public CorrelationIdGenerator getCorrelationIdGenerator() {
```

Comment:
This is needed by the `SphereClientConfigBuilder`, but I reduced the visibility to default.